source/includes/introduction.md

```diff
@@ -1,9 +1,21 @@
 # Introduction
 
-Welcome to the Polymer public API documentation!
+Introducing the Revolutionary Polymer public API Documentation!
```

Comment:
I feel all the changes here are gimmicky as well. Our target group is engineers, not teenagers.